### ISSUE TITLE
ImportError: No module named 'constants'

### ISSUE THREAD

**Issue by vbraun** [nir0s/ld#1]
This is with pip-installed ld-0.1.1:
~~~~
>>> import ld
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vbraun/Code/binary-pkg/tools/binary-pkg/lib/python3.4/site-packages/ld/__init__.py", line 3, in <module>
    import constants as const
ImportError: No module named 'constants'
~~~~

**Comment by nir0s** [comment]
fixed in 0.1.2. ld is now tested on Python 3.5 as well.

**Comment by vbraun** [comment]
I can confirm that it works, thanks!
